lang/rs/bindings/cners/examples/loopback/run.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Build CNDP and install (sudo CNE_DEST_DIR=/ make install) before running this script.
+# Usage: ./run.sh <json config file> <port name> <core id (optional)>
+# eg: ./run.sh ../../fwd.json enp134s0:0 25
+
+# JSON file
+CONFIG=${1:-"../../fwd.json"}
+
+# Lport
+PORT=${2:-"enp134s0:0"}
```

Comment:
We already have the port name in the json file, lets not duplicate it here too. Cores are also defined in the json file.

Reply:
All the ports in the json file should be used for send/recv, right? I agree with Keith, why would somebody put something in the json file they do not want to use?

Reply:
In JSON file, user can give all ports. But this simple example application will send/recv only from one port at a time. So in command line user has to specify either port id or port name from the list. Keith and I just talked. We agreed to replace port name with port id since port name can change across systems.

Reply:
Regarding affinitizing thread to core, Rust applications mostly uses it's own thread model and library to create and manage thread and doesn't use CNE thread APIs. So currently the "threads" section in JSON file is unused. So after Rust applications creates loopback thread, to get best performance (similar to C CNDP application) it can (optionally) set core affinity.  But we can certainly provide a Rust API to assign an lcore-group to Rust thread and there by setting core affinity to that thread based on configuration in JSON file. Just that I've not implemented it yet.

Reply:
Also the thread section also helps the application define what a thread is doing in the application. I know this does not apply the Rust threads, but we can not forget this point. Also the thread section also assigns lport to threads or functions i.e. fwd:0, fwd:1 how do we arrange the lports to groups of threads in Rust?

Reply:
Rust applications using CNE Rust library can create Rust threads using standard routines and register those threads with CNE and we can send/recv packets using lports from that threads. The synchronization/thread safety when using same lports from multiple threads is taken care by Rust CNE library.  The loopback sample application currently doesn't use Threads section defined in JSONC file. Not sure if that answer above question. If not, I've not understood the requirement well. We can discuss in a call.